<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>faradei</class>
 <widget class="QWidget" name="faradei">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="picture">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>761</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="picturebox" name="pb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>761</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="return_to">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>560</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Вернуться</string>
   </property>
  </widget>
  <widget class="QComboBox" name="R_change">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>560</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QComboBox" name="B_change">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>161</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QComboBox" name="B_change_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>161</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>picturebox</class>
   <extends>QGraphicsView</extends>
   <header location="global">picturebox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
